docs/shared/reference/environment-variables.md

```diff
@@ -14,6 +14,7 @@ The following environment variables are ones that you can set to change the beha
 | NX_PROFILE                       | string  | Prepend `NX_PROFILE=profile.json` before running targets with Nx to generate a file that be [loaded in Chrome dev tools](/recipes/troubleshooting/performance-profiling) to visualize the performance of Nx across multiple processes. |
 | NX_PROJECT_GRAPH_CACHE_DIRECTORY | string  | The project graph cache is stored in `node_modules/.cache/nx` by default. Set this variable to use a different directory.                                                                                                              |
 | NX_PROJECT_GRAPH_MAX_WORKERS     | number  | The number of workers to use when calculating the project graph.                                                                                                                                                                       |
+| NX_PARALLEL                      | number  | The number of tasks Nx should run in parallel. Defaults to 3, overridable in nx.json                                                                                                                                                   |
```

Comment:
Surely the env var should have a higher priority than the static config value?

Reply:
We should also document that the env var will have no effect if the parallel flag is set to a number already

Reply:
Thinking out loud I wonder if we should throw if both are set? Seems like it would be unintentional?

Reply:
I don't think it makes sense - and on review I was wrong when I wrote this bit of docs. The env var would have higher precedence as it is currently written.